.circleci/monorepo.sh

```diff
@@ -201,11 +209,15 @@ function get_builds {
 
 function debug {
   echo -e "\n\nDEBUG INFORMATION"
-  echo -e "\n\n=== Branches ==="
-  cat "${TMP_DIR}/branches.txt"
+  if [[ -r "${TMP_DIR}/branches.txt" ]]; then
```

Comment:
Saw errors cat'ing these files when debug was enabled and specified tag, cause these files are created.